### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
```
"autodappsvalidation.online",
"dapp-vadilate.tools",
"dappconnectwallet.web.app",
"dapppsconnect-web.com",
"dappreconnection.art",
"dapps-connect-link.com",
"dapps-walletconnectprotocol.link",
"dappsbridgelink.com",
"dappsconnect.xyz",
"dappsnodes.org",
"dappssuports.online",
"dappwebwallets.com",
"lacasadepapeltoken.org",
"metaupgraden.com",
"presale.shibachi.site",
"revalidateconnect.site",
"shibaalpha.com",
"shibainu-giftorg",
"spikeyspacefish.com",
"sync-contract.com",
"thespacebulls.shop",
"walletconnects.com",
"web.dappsconnect.xyz",
"web3alldapprotocol.com",
```
Will correct the final list after the duplicates check is complete.

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
removed
"autodappsvalidation.online",
"dapp-vadilate.tools",
"dappconnectwallet.web.app",
"dapps-walletconnectprotocol.link",
"dappssuports.online",
"lacasadepapeltoken.org",
"metaupgraden.com",
"revalidateconnect.site",
"shibaalpha.com",
"spikeyspacefish.com",
"thespacebulls.shop",

File: all.json [hunk 1]
```diff
@@ -7,6 +7,19 @@
...
     "16-b.it",
+		"dapppsconnect-web.com",
+		"dappreconnection.art",
+		"dapps-connect-link.com",
+		"dappsbridgelink.com",
+		"dappsconnect.xyz",
+		"dappsnodes.org",
+		"dappwebwallets.com",
+		"presale.shibachi.site",
+		"shibainu-giftorg",
```
Comment (jacogr) [inline all.json:18]:
```suggestion
		"shibainu-gift.org",
```

**Comment by dubstard** [comment]
ahh looks like i added the org one in https://github.com/polkadot-js/phishing/pull/1062
but it didn't pop in the initial dupes check due to the missing dot before the GTLD
dang it, so sorry

**Review by dubstard — COMMENTED** [review]
"shibainu-gift.org", was already there via PR 1061, sorry

**Review by dubstard — COMMENTED** [review]
ok, last fix for this PR, sheesh 😊

**Review by jacogr — APPROVED** [review]

**Pull request merged**